pennylane/ops/qubit.py

```diff
@@ -455,21 +478,40 @@ class CRX(Operation):
     num_wires = 2
     par_domain = "R"
     grad_method = "A"
-    generator = [np.array([[0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 1], [0, 0, 1, 0]]), -1 / 2]
+    generator = [
+    np.array([[0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 1], [0, 0, 1, 0]]), -1 / 2]
```

Comment:
This line reformat looks a bit weird?

Reply:
Yes, I tried out an autoformatter here but will be reverting the changes